frontend/scss/components/molecules/input-bar.scss

```diff
@@ -108,19 +108,19 @@
       transform: rotate(-90deg);
       height: 20px;
       width: 20px;
-      fill: color('white');
+      fill: safeColor('white');
 
       @media (min-width: 600px) {
         display: none;
       }
     }
 
     &[disabled] {
-      background: color('boulder');
+      background: color('silver');
```

Comment:
Is `silver` or `mercury` the light grey of choice? Or are we using three different shades of grey?

Reply:
Also I thought we were not visually disabling the input button anymore?